bitmind/validator/forward.py

```diff
@@ -146,7 +146,7 @@ async def forward(self):
         performance_tracker=self.performance_tracker)
     
     # Logging image source and verification details
-    source_name = wandb_data['model'] if 'model' in wandb_data else wandb_data['dataset']
+    source_name = wandb_data['model'] if 'model' in wandb_data else wandb_data['source_dataset']
```

Comment:
Q: When would wandb_data contain 'model'? Could use a comment explaining the logic here and why we want source_dataset in some cases

Reply:
Will update with a comment explaining the logic!